prometheus_nats_exporter.go

```diff
@@ -118,6 +118,7 @@ func main() {
 	flag.StringVar(&opts.HTTPUser, "http_user", "", "Enable basic auth and set user name for HTTP scrapes.")
 	flag.StringVar(&opts.HTTPPassword, "http_pass", "", "Set the password for HTTP scrapes. NATS bcrypt supported.")
 	flag.StringVar(&opts.Prefix, "prefix", "", "Set the prefix for all the metrics.")
+	flag.BoolVar(&opts.UseOldServerID, "use_old_server_id", false, "Disables using ServerID from /varz")
```

Comment:
This could break existing dashboards and filters.  Would it be better to go the opposite direction and have a flag that says "use_internal_server_id" - or something like that?